invenio_app_rdm/static_pages/static_pages.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio App RDM is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Static pages."""
+import os
+from pathlib import Path
+
+from flask_webpackext import current_webpack
+from invenio_db import db
+from invenio_pages import Page
+
+
+class StaticPages:
+    """Static pages."""
+
+    def page_data(self, page):
+        """Returns content of template."""
+        assets_path = current_webpack.project.path
+        overriden_page = Path(
+            assets_path + os.path.join("/templates/static-pages", page)
+        )
+        if overriden_page.exists():
+            return (
+                Path(assets_path + os.path.join("/templates/static-pages", page))
+                .read_bytes()
+                .decode("utf8")
+            )
+
+        return (
+            Path(Path(__file__).parent.as_posix() + os.path.join("/templates", page))
+            .read_bytes()
+            .decode("utf8")
+        )
+
+    def run(self, force=False):
+        """Run the creation of Static Pages."""
+        pages = [
+            Page(
+                url="/about",
+                title="About",
+                description="About",
+                content=self.page_data("about.html"),
+                template_name="invenio_pages/default.html",
+            ),
+            Page(
+                url="/contact",
+                title="Contact",
+                description="Contact",
+                content=self.page_data("contact.html"),
+                template_name="invenio_pages/default.html",
+            ),
+            Page(
+                url="/guide/search",
```

Comment:
minor: one feature we lose here is [the l10n](https://github.com/inveniosoftware/invenio-app-rdm/blob/3cff73b4e3726f33122176368213b662e77d7571/invenio_app_rdm/theme/views.py#L69-L72) of the help search guide, which could be achieved by dropping e.g. a `invenio_app_rdm/help/search.es.html` file to define the Spanish translation.

This might apply though in general to pages, so we might revisit in the future to see how one could provide other languages for a single page.